ui/edit_profile.html

```diff
@@ -48,7 +48,8 @@
                 </tr>
                 <tr>
                     <td><label> <h4><b><font color="white">Gender:</font></b> </label></td></h4>
-                    <td><h4><input type="text" name="gender" value="{{pcuser.gender}}"></h4></td>
+                    <td><h4><select name="gender"><option value="" selected disabled>Select</option>
+                    <option value="Male">Male</option><option value="Female">Female</option> </select></h4></td>
```

Comment:
There is a lot of talk these days on diversity Inclusive forms. 
My recommendation would be to make either gender information an optional input  but if that is not possible then add one more choice "decline to state"